.github/workflows/appstore-build-publish.yml

```diff
@@ -10,7 +10,7 @@ on:
     types: [published]
 
 env:
-  PHP_VERSION: 7.4
+  PHP_VERSION: 8.1
```

Comment:
Could be breaking, please advise

Reply:
ack, possibly breaking, we need 7.4 yet.